examples/vercel-ai-chatbot/lib/hooks/use-vellum-chat.tsx

```diff
@@ -32,6 +33,20 @@ const useVellumChat = ({
   const triggerRequest = React.useCallback(
     async (request: { messages: ChatMessage[]; id: string }) => {
       setIsLoading(true)
+      if (!path.includes('chat')) {
+        const userMessage = request.messages.find(
+          m => m.role === 'USER' && m.content?.type === 'STRING'
+        )
+        const defaultChatTitle =
+          (userMessage?.content?.value as string)?.slice(0, 10) ?? 'New chat'
+
```

Comment:
oos but open to ideas - better default title logic

Reply:
Feels aligned with OpenAI chats which I think is good 👍